plugins/env/app/helpers/environment_variables_helper.rb

```diff
@@ -0,0 +1,9 @@
+module EnvironmentVariablesHelper
```

Comment:
Will do. Wasn't sure if its more view or model logic... can we keep the functionality to decorate controllers still?